functional_tests/bin/wait-for-it.sh

```diff
@@ -0,0 +1,182 @@
+#!/usr/bin/env bash
```

Comment:
Is `wait-for-it.sh` much different than our use of [`takis`](https://github.com/mozilla/normandy/blob/645bcc4f4cfe9bbb472704162c8b543624c90803/recipe-server/bin/ci/test#L11)? Would using that instead be easier?